wweval/R/ms_fig5.R

```diff
@@ -216,6 +278,78 @@ make_fig5_hub_performance <- function(all_scores,
   return(p)
 }
 
+make_fig5_density <- function(all_scores,
+                              analysis_type,
+                              models_to_show,
+                              baseline_model = "COVIDhub-4_week_ensemble") {
+  subset_scores <- all_scores |>
+    dplyr::filter(model %in% !!models_to_show)
+
+  baseline_scores <- subset_scores |>
+    dplyr::filter(model == {{ baseline_model }}) |>
+    dplyr::select(
+      location, forecast_date, target_end_date,
+      horizon, interval_score
+    ) |>
+    dplyr::group_by(location, forecast_date) |>
+    dplyr::summarize(mean_baseline_score = mean(interval_score))
+
+  scores_final <- subset_scores |>
+    dplyr::group_by(
+      forecast_date, location, model
+    ) |>
+    dplyr::summarize(mean_wis = mean(interval_score)) |>
+    dplyr::left_join(baseline_scores, by = c(
+      "forecast_date", "location"
+    )) |>
+    dplyr::mutate(relative_wis = mean_wis / mean_baseline_score) |>
+    dplyr::filter(
+      model != {{ baseline_model }},
+      !is.na(relative_wis)
+    )
+
+
+  colors <- plot_components()
+
+  p <- ggplot(scores_final) +
+    tidybayes::stat_histinterval(
+      aes(
+        x = model, y = relative_wis + 1e-8,
```

Comment:
the relative wis shouldn't be zero anywhere. Why is this needed?

Reply:
I can remove, might have been a holdover from a previous plot